### PR TITLE
Fixes #25623: Missing documentation on rudder.users.cleanup.account.disableAfterLastLogin

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/25623

### CODE REVIEW

**Comment by ElaadF** [comment]
Commit modified

File: src/reference/modules/administration/pages/security.adoc [hunk 1]
```diff
@@ -34,6 +34,13 @@ rudder.server.hstsIncludeSubDomains=true
 rudder.auth.idle-timeout=15 minutes
 ----
 
+* When a user have not logged in, by default for 90 days, the account is marked as *disabled*, you can change this value by modifying or adding the parameter `rudder.users.cleanup.account.disableAfterLastLogin` in `/opt/rudder/etc/rudder-web.properties` by following this format : https://scala-lang.org/api/3.x/scala/concurrent/duration/Duration$.html#apply-537
+
+* A user can be *deleted* after a certain period of time, by default the parameter `rudder.users.cleanup.account.deletedAfterLastLogin` is set to 120 days. You can change this value by modifying or adding this parameter in `/opt/rudder/etc/rudder-web.properties` by following this format : https://scala-lang.org/api/3.x/scala/concurrent/duration/Duration$.html#apply-537
```
Comment (ncharles) [inline src/reference/modules/administration/pages/security.adoc:39]:
```suggestion
* A user is *deleted* after a certain period of time, by default the parameter `rudder.users.cleanup.account.deletedAfterLastLogin` is set to 120 days. You can change this value by modifying or adding this parameter in `/opt/rudder/etc/rudder-web.properties` by following this format : https://scala-lang.org/api/3.x/scala/concurrent/duration/Duration$.html#apply-537
```

Comment (clarktsiory) [reply src/reference/modules/administration/pages/security.adoc:39]: I don't think the user wants to open the scaladoc, we can hardcode the format, and users will not likely want anything else than days, that leaves : `d, day`

File: src/reference/modules/administration/pages/security.adoc [hunk 2]
```diff
@@ -34,6 +34,13 @@ rudder.server.hstsIncludeSubDomains=true
...
+
+* When an user is deleted (for example removed from `rudder-user.xml` file), then the user information, like status changes, are not purged immediately (typically to be able to do post-deletion accountability).
+You can modify the period of time (by default 30 days) after which the purge is performed by editing or adding the parameter `rudder.users.cleanup.purgeDeletedAfter` and by following this format : https://scala-lang.org/api/3.x/scala/concurrent/duration/Duration$.html#apply-537
```
Comment (clarktsiory) [inline src/reference/modules/administration/pages/security.adoc:42]: Same for the scaladoc

**Comment by ElaadF** [comment]
Commit modified

**Review by clarktsiory — APPROVED** [review]

**Comment by ElaadF** [comment]
OK, merging this PR

**Pull request merged**